bootstrap/ansible/settings_template.tmpl

```diff
@@ -43,6 +45,9 @@ DEFAULT_FROM_EMAIL = EMAIL_FROM
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+#        'OPTIONS': {
+#                'options': '-c search_path={{ schema_name }}'
```

Comment:
This variable isn't defined, causing the playbook to fail.

Reply:
Yeah, I had it for a bit when I was trying to fix PG 15, but another fix worked better. I'll remove the comments